source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -583,6 +594,17 @@ export default class NetworkStatusStore extends Store {
     openLocalDirectoryChannel.send(path);
   }
 
+  convertBytesToSize = (bytes: number): string => {
```

Comment:
@DeeJayElly I feel like this method belongs to utils rather then here. Can you please check if we already have a similar one? I am pretty sure we do...

Reply:
Done